lti_consumer/templates/email/user_score_notification.html

```diff
@@ -0,0 +1,141 @@
+%>
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <title lang="en">${_("OpenCV")}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+
+    <style type="text/css">
+        @media only screen and (min-device-width: 601px) {
+            .content {
+                width: 600px !important;
+            }
+        }
+
+        @-ms-viewport {
+            width: device-width;
+        }
+
+        /* Column Drop Layout Pattern CSS */
+        @media only screen and (max-width: 450px) {
+            td[class="col"] {
+                display: block;
+                width: 100%;
+                -moz-box-sizing: border-box;
+                -webkit-box-sizing: border-box;
+                box-sizing: border-box;
+                float: left;
+                text-align: left !important;
+                padding-bottom: 20px;
+            }
+        }
+    </style>
+</head>
+<body>
+<div lang="en" style="
+    display:none;
+    font-size:1px;
+    line-height:1px;
+    max-height:0px;
+    max-width:0px;
+    opacity:0;
+    overflow:hidden;
+    visibility:hidden;
+"></div>
+
+<div bgcolor="#f5f5f5" lang="en" dir="ltr" style="
+    margin: 0;
+    padding: 0;
+    min-width: 100%;
+">
+    <!-- Hack for outlook 2010, which wants to render everything in Times New Roman -->
+    <!--[if mso]>
+    <style type="text/css">
+        body, table, td {
+            font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif !important;
+        }
+    </style>
+    <![endif]-->
+
+    <!--[if (gte mso 9)|(IE)]>
+    <table role="presentation" width="600" align="center" cellpadding="0" cellspacing="0" border="0">
+        <tr>
+            <td>
+    <![endif]-->
+
+    <!-- CONTENT -->
+    <table class="content" role="presentation" align="center" cellpadding="0" cellspacing="0" border="0"
+           width="100%" style="
+        font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+        font-size: 1em;
+        line-height: 1.5;
+        max-width: 600px;
+        padding: 0 20px 0 20px;
+    ">
+        <tr>
+            <!-- MAIN -->
+            <td class="main" bgcolor="#ffffff" style="padding: 30px 20px;">
+
+                <table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+                    <tr>
+                        <td width="70">
+                            <a href="${lms_root}"><img src="${site_logo}" width="90" height="90" alt="Go to Your ${platform_name} Home Page"/></a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>
+                            <h1>${_("Review available")}</h1>
+                            <p style="color: rgba(0,0,0,.75);">
+                                % if user_first_name:
+                                Dear ${user_first_name}
```

Comment:
I would use condition case only for sensitive part like `${user_first_name}`

Reply:
Done